test/Microsoft.TemplateEngine.Authoring.CLI.IntegrationTests/TestTemplate/.template.config/template.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
should we add it to `TestTemplates` project instead to unify location of all test templates?